subsys/net/openthread/Kconfig.defconfig

```diff
@@ -0,0 +1,149 @@
+# Copyright (c) 2021 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# The purpose of this file is to define new default values of settings related
+# to building Thread from sources.
+# This file only changes defaults and thus all symbols here must be prompltless
+# and safeguarded so that they only are applied when building Thread.
+
+if NET_L2_OPENTHREAD
+
+choice LIBC_IMPLEMENTATION
+	default NEWLIB_LIBC
+endchoice
+
+config NUM_METAIRQ_PRIORITIES
+	int
+	default 1
+
+config SYSTEM_WORKQUEUE_STACK_SIZE
+	int
+	default 1120
+
+config HW_STACK_PROTECTION
+	bool
+	default y
+
+config NET_CONNECTION_MANAGER
+	bool
+	default y
+
+config MAIN_STACK_SIZE
+	int
+	default 2048
+
+config INIT_STACKS
+	bool
+	default y
+
+config NUM_METAIRQ_PRIORITIES
+	int
+	default 1
+
+config NET_LOG
+	bool
+	default y
+
+config NET_STATISTICS
+	bool
+	default y
+
+config NET_IPV6_NBR_CACHE
+	bool
+	default n
+
+config NET_IPV6_MLD
+	bool
+	default n
+
+config NET_TX_STACK_SIZE
+	int
+	default 1200
+
+config NET_RX_STACK_SIZE
+	int
+	default 1500
+
+config NET_PKT_RX_COUNT
+	int
+	default 10
+
+config NET_PKT_TX_COUNT
+	int
+	default 16
+
+config NET_BUF_RX_COUNT
+	int
+	default 16
+
+config NET_BUF_TX_COUNT
+	int
+	default 16
+
+config OPENTHREAD_MANUAL_START
+	bool
+	default y
+
+choice OPENTHREAD_SECURITY
+	default OPENTHREAD_NRF_SECURITY_CHOICE
+endchoice
+
+config MBEDTLS_CIPHER_MODE_CBC
+	bool
+	default n
+
+config MBEDTLS_CIPHER_MODE_CTR
+	bool
+	default n
+
+config MBEDTLS_CIPHER_MODE_CFB
+	bool
+	default n
+
+config MBEDTLS_CIPHER_MODE_OFB
```

Comment:
From here and below I have not verified the symbols, could you please do that yourself.

Reply:
# This is needed to properly handle MPSL in openthread samples

Reply:
That is not what I meant with that comment.

What I meant was that I had not been through all the symbols to see if those are already unavailable or default `n` in a default setup.

I found several symbols that already defaulted to the desired value, if you check the comments above, but asked you to carry out the same check for symbols below.